### PR TITLE
Update PvD section

### PR DESCRIPTION
**Pull request by klarose**
We were incorrectly describing the behaviour of the capport PvD option.
Update the text to better reflect how it will work, and reference the
new draft describing that behaviour.

This removes most of the section describing the behaviour, since it's now in the new draft.

Fixes Issue #17.

### CODE REVIEW

**Comment by martinthomson** [comment]
Let's talk about adoption of the capport-pvd draft before we make this sort of change.

**Comment by klarose** [comment]
Martin, was the capport-pvd draft discussed during IETF 104? I don't see anything in the minutes.

**Comment by tfpauly** [comment]
@klarose we did not specifically discuss the capport-pvd draft. I think that doesn't need to be in this first core flight of docs. Note we'll be discussing the core PvD doc more in intarea and hopefully will get that out to last call.

**Comment by klarose** [comment]
@tfpauly Alright. The existing content is still wrong, though. So, to fix this, perhaps we should just remove the references to PvDs in general?

File: draft-ietf-capport-architecture.xml [hunk 1]
```diff
@@ -293,37 +293,16 @@
...
-           key-value pairs to be downloaded over HTTPS.
-           This JSON file would fill the role of the Captive Portal API
+           Bootstrap Information containing the URI for the JSON-based API
            described in <xref target="section_api"/>.
          </t>
          <t>
            The PvD security model provides secure binding between the
            information provided by the trusted Router Advertisement and the
            HTTPS server.
          </t>
```
Comment (martinthomson) [inline draft-ietf-capport-architecture.xml]: This paragraph might also go away.

**Pull request merged**